app/views/content/home.md

```diff
@@ -5,10 +5,10 @@
     Read official Department for Education information to help you become a teacher. Learn about funding, teacher training, and steps to become a teacher. 
   date: "2021-06-16"
   fullwidth: true
-  subtitle: "Whether you're ready to start your career in teaching or just curious, we can answer your questions at a Get Into Teaching event this November."
-  subtitle_button: "Get Into Teaching events"
-  subtitle_link: "/events/about-get-into-teaching-events"
-  image: "media/images/content/hero-images/event-image.jpg"
+  subtitle: "Whether you're just thinking about teaching or you're ready to apply, get tailored advice and insights sent straight to your inbox."
```

Comment:
Is "insights" the right word here - doesn't sound quite as appealing as it perhaps could? Or should we just remove it - this is quite a long sentence compared to what we used to have?

(but this is very minor and definitely shouldn't hold up merging tomorrow!)